.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### What?
+
+<!-- What changes are made? -->
+<!-- (if there are many significant changes, a list might be a good format)  -->
+
+<br>
```

Comment:
I feel like i'm just gonna erase these tags every time, can we remove them? Or nah?

Reply:
True, since you will have to edit it anyway maybe remove?

Reply:
I mean they're not visible so you don't _have_ to remove them, but sure. Would you like to remove all of the comments? I think the last one that tells you to add screenshots/GIF's might serve a purpose, no?

Reply:
I ment the `<br>` tag - sorry if that was unclear. The comments i think make sense

Reply:
Aha, sorry, I was confused. 🤕 
Personally I like the extra space/air in between the sections but if you guys don't want it I can of course remove it.